linode/helper/frameworkfilter/filter.go

```diff
@@ -157,6 +167,17 @@ func (f Config) GetAndFilter(
 	if d != nil {
 		return nil, d
 	}
-
 	return locallyFilteredElements, nil
 }
+
+func FilterTypeString(value string) (interface{}, error) {
```

Comment:
nitpick: We can use `any` here rather than `interface{}` to make the code a tiny bit more readable